app/adapters/institutional-user.ts

```diff
@@ -0,0 +1,10 @@
+import OsfAdapter from './osf-adapter';
+
+export default class InstitutionalUserAdapter extends OsfAdapter {
+}
```

Comment:
Since `institutional_users` is not a top level endpoint, but under `/institutions/:institution_id/users`, we should probably customize the adapter for that url.

Reply:
We have a nice way of doing this. Here's an example:
https://github.com/CenterForOpenScience/ember-osf-web/blob/7ee8ac9affb0a01d467d9e248733d6e166164acc/app/adapters/draft-registration.ts#L3-L5
This tells our custom adapter which of this model's relationships is its parent and to use that relationship's meta to determine where the the endpoint for this model lives. So, we should add a relationship back to the institution and specify that here.

Reply:
Oh, you already got it! :100: